### PR TITLE
Update Upsell to New Design

### PR DESCRIPTION
**Pull request by jbuckner**
This updates the design and adds a dev menu so you can display it on demand.

Screen Shot 2020-11-11 at 10 48 57 AM

### CODE REVIEW

File: packages/donation-form/test/tests/modals/upsell-modal-content.test.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
+import { html, fixture, expect, oneEvent } from '@open-wc/testing';
+import { UpsellModalContent, UpsellModalCTAMode } from '../../../src/modals/upsell-modal-content';
+import '../../../src/modals/upsell-modal-content';
+
+describe('Upsell Modal Content', () => {
```
Comment (iisa) [inline packages/donation-form/test/tests/modals/upsell-modal-content.test.ts:5]: thank you!

File: packages/donation-form/src/payment-flow-handlers/handlers/paypal-flow-handler.ts [hunk 2]
```diff
@@ -284,10 +284,10 @@ export class PayPalFlowHandler
     const upsellButtonDataSource = await handler?.renderPayPalButton({
       selector: '#paypal-upsell-button',
       style: {
-        color: 'gold' as paypal.ButtonColorOption, // I'm not sure why I can't access the enum directly here.. I get a UMD error
+        color: 'blue' as paypal.ButtonColorOption,
         label: 'paypal' as paypal.ButtonLabelOption,
         shape: 'rect' as paypal.ButtonShapeOption,
-        size: 'medium' as paypal.ButtonSizeOption,
+        size: 'responsive' as paypal.ButtonSizeOption,
```
Comment (iisa) [inline packages/donation-form/src/payment-flow-handlers/handlers/paypal-flow-handler.ts:290]: q: are these configurations that we send to paypal?

**Review by iisa — APPROVED** [review]
👍 looks good. just had a couple of questions. thanks for the bulk tests and docs.

File: packages/donation-form/src/payment-flow-handlers/handlers/paypal-flow-handler.ts (hunk 2 above)
Comment (jbuckner) [reply packages/donation-form/src/payment-flow-handlers/handlers/paypal-flow-handler.ts:290]: Yep! These get passed through to the PayPal button renderer.

**Pull request merged**